src/components/PageHeading.vue

```diff
@@ -15,19 +15,14 @@
 </template>
 
 <script lang="ts" setup>
-  import { PBreadCrumbs, BreadCrumbs, Size } from '@prefecthq/prefect-design'
-  import { PropType, computed } from 'vue'
+  import { PBreadCrumbs, BreadCrumbs } from '@prefecthq/prefect-design'
+  import { computed } from 'vue'
+  import { Size } from '@/types'
 
-  const props = defineProps({
-    crumbs: {
-      type: Array as PropType<BreadCrumbs>,
-      required: true,
-    },
-    size: {
-      type: String as PropType<Size>,
-      default: 'xl',
-    },
-  })
+  const props = defineProps<{
+    crumbs: BreadCrumbs,
+    size?: Size,
```

Comment:
Before there was a default of `xl` which I don't think is accounted for now.

Reply:
![image](https://github.com/PrefectHQ/prefect-ui-library/assets/6776415/8863b9a6-53cf-45cb-a65f-e185e8e0749b)